pkg/common/configs/configvalidator.go

```diff
@@ -18,6 +18,7 @@ package configs
 
 import (
     "fmt"
+    "github.com/cloudera/yunikorn-core/pkg/cache"
```

Comment:
This seems to introduce cyclic dependency, it could not build.

Reply:
removed all cyclic dependencies for policies.